modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaUpdateFileEntryMethodCheck.java

```diff
@@ -57,16 +57,25 @@ protected String format(
 				String variableName = getVariableName(methodCall);
 
 				if (!hasClassOrVariableName(
-						"DLAppLocalService", content, content, methodCall) &&
+						"DLAppLocalService", javaMethodContent, content,
+						fileName, methodCall) &&
 					!variableName.contains("DLAppLocalServiceUtil")) {
 
 					continue;
 				}
 
 				String className = getVariableTypeName(
-					newContent, newContent, parameterList.get(8), true);
+					javaMethodContent, newContent, fileName,
+					parameterList.get(8), true);
 
-				if (!className.equals("byte[]")) {
+				if ((className == null) || !className.equals("byte[]")) {
+					continue;
+				}
+
+				String parameterClass = getVariableTypeName(
+					javaMethodContent, content, fileName, parameterList.get(7));
+
+				if (parameterClass == null) {
```

Comment:
@TamyrisTorres, can we rename the variable `className` by `variableTypeName` and reuse this variable instead of creating the new variable `parameterClass`?

It's important to keep consistency, since Brian probably will complain about it.